### PR TITLE
doc(lxy): add gumbel muzero to readme

### PR DESCRIPTION
**Pull request by karroyan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,22 +110,21 @@ LightZero is a library with a [PyTorch](https://pytorch.org/) implementation of
 - [MuZero](https://arxiv.org/abs/1911.08265)
```
Comment (PaParaZz1) [inline README.md:110]: you need to also add these information in Chinese readme

File: README.md [hunk 2]
```diff
@@ -110,22 +110,21 @@ LightZero is a library with a [PyTorch](https://pytorch.org/) implementation of
...
+| ------------- | --------- | ------ | ------------- | --------------------- | ------------- |
+| Atari         | ---       | ✔    | ✔           | ✔                   | ✔             |
+| tictactoe     | ✔       | ✔    | 🔒          | 🔒                  | ✔             |
+| gomoku        | ✔       | ✔    | 🔒          | 🔒                  | ✔             |
+| chess         | 🔒       | 🔒   | 🔒          | 🔒                  | 🔒             |
+| go            | 🔒       | 🔒   | 🔒          | 🔒                  | 🔒             |
+| lunarlander | ---       | ✔    | ✔           | ✔                   | ✔             |
+| bipedalwalker   | ---       | ✔    | ✔           | ✔                   | 🔒             |
+| cartpole     | ---       | ✔    | ✔           | ✔                   | ✔             |
+| pendulum      | ---       | ✔    | ✔           | ✔                   | 🔒             |
```
Comment (PaParaZz1) [inline README.md:127]: add repo `mctx` in Acknowledgments

File: README.md [hunk 3]
```diff
@@ -110,22 +110,21 @@ LightZero is a library with a [PyTorch](https://pytorch.org/) implementation of
...
```
Comment (PaParaZz1) [inline README.md:128]: add some benchmark results about gumbel muzero

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -110,22 +110,21 @@ LightZero is a library with a [PyTorch](https://pytorch.org/) implementation of
...
-| pendulum      | ---       | ✔    | ✔           | ✔                   |
+| Env./Alg.      | AlphaZero | MuZero | EfficientZero | Sampled EfficientZero | Gumbel MuZero |
+| ------------- | --------- | ------ | ------------- | --------------------- | ------------- |
+| Atari         | ---       | ✔    | ✔           | ✔                   | ✔             |
+| tictactoe     | ✔       | ✔    | 🔒          | 🔒                  | ✔             |
+| gomoku        | ✔       | ✔    | 🔒          | 🔒                  | ✔             |
+| chess         | 🔒       | 🔒   | 🔒          | 🔒                  | 🔒             |
+| go            | 🔒       | 🔒   | 🔒          | 🔒                  | 🔒             |
+| lunarlander | ---       | ✔    | ✔           | ✔                   | ✔             |
+| bipedalwalker   | ---       | ✔    | ✔           | ✔                   | 🔒             |
```
Comment (puyuan1996) [inline README.md:125]: It would be a good idea to add bipedalwalker and pendulum gumbel related config

File: README.md [hunk 5]
```diff
@@ -199,6 +198,14 @@ Below are the benchmark results of [AlphaZero](https://github.com/opendilab/Ligh
...
+<p align="center">
+  <img src="assets/benchmark/ablation/pong_gmz_ns.png" alt="Image Description 1" width="23%" height="auto" style="margin: 0 1%;">
+  <img src="assets/benchmark/ablation/mspacman_gmz_ns.png" alt="Image Description 2" width="23%" height="auto" style="margin: 0 1%;">
+  <img src="assets/benchmark/ablation/gomoku_gmz_ns.png" alt="Image Description 3" width="23%" height="auto" style="margin: 0 1%;">
+  <img src="assets/benchmark/ablation/lunarlander_gmz_ns.png" alt="Image Description 3" width="23%" height="auto" style="margin: 0 1%;">
```
Comment (puyuan1996) [inline README.md:206]: It appears that the Gumbel benchmark image was not uploaded correctly. Consider using a command like 
`git add -f assets/benchmark/ablation/pong_gmz_ns.png ` to force the addition of the image.

File: README.md [hunk 6]
```diff
@@ -214,6 +221,7 @@ The following are the detailed paper notes (in Chinese) of the above algorithms:
 - [MuZero](https://github.com/opendilab/LightZero/blob/main/assets/paper_notes/MuZero.pdf)
 - [EfficientZero](https://github.com/opendilab/LightZero/blob/main/assets/paper_notes/EfficientZero.pdf)
 - [SampledMuZero](https://github.com/opendilab/LightZero/blob/main/assets/paper_notes/SampledMuZero.pdf)
+- [GumbelMuZero](https://github.com/opendilab/LightZero/blob/main/assets/paper_notes/GumbelMuZero.pdf)
```
Comment (puyuan1996) [inline README.md:224]: GumbelMuZero paper notes is not uploaded correctly.

**Review by puyuan1996 — CHANGES_REQUESTED** [review]

**Pull request merged**